pages/en/lb3/Installing-the-Oracle-connector.md

```diff
@@ -11,24 +11,36 @@ summary:
-At runtime, strong-oracle also requires dynamic libraries from 
-[Oracle Database Instant Client](http://www.oracle.com/technetwork/database/features/instant-client/index.html).
-
-The [LoopBack Oracle Installer](https://github.com/strongloop/loopback-oracle-installer) module takes care of the binary dependencies and simplifies the whole process.  
-The LoopBack Oracle installer downloads and extracts the prebuilt LoopBack Oracle binary dependencies into the parent module's `node_modules` directory and sets up the environment for the [Oracle Database Instant Client](http://www.oracle.com/technetwork/database/features/instant-client/index.html).
-
 To install the Oracle connector, use the following command:
 
-```shell
+```
 $ npm install loopback-connector-oracle --save
 ```
 
-{% include important.html content="
 
-If you need to use the Oracle driver directly, see [https://github.com/strongloop/strong-oracle](https://github.com/strongloop/strong-oracle)
+The Oracle connector version 3.0.0 or above depends on the [oracledb](https://github.com/oracle/node-oracledb) module as the Node.js driver for Oracle databases.  Since `oracledb` is a [C++ add-on](http://nodejs.org/api/addons.html), the installation requires the presence of C++ development tools to compile and build the module from source code. At runtime, `oracledb` also requires dynamic libraries from [Oracle Database Instant Client](http://www.oracle.com/technetwork/database/features/instant-client/index.html).
 
-" %}
+The [LoopBack Oracle Installer](https://github.com/strongloop/loopback-oracle-installer) module takes care of the binary dependencies and simplifies the whole process. The installer automatically downloads and extracts the prebuilt `oracledb` to /loopback-connector-oracle/node_modules directory and Oracle Database Instant Client into `<UserHomeDirectory>/oracle-instant-client`. 
+If a prebuilt bundle does not exist for your client platform and node version, the installer prompts the user to install oracle pre-requisites offline following instructions for their OS in https://github.com/oracle/node-oracledb/blob/master/INSTALL.md. 
+
+Additional steps on OS X: 
+```
+ln -s libocci.dylib.12.1 libocci.dylib
+export OCI_INCLUDE_DIR=~/instantclient_11_2/sdk/include
+```
+Once oracle pre-requisites are installed, user needs to re install loopback-connector-oracle node module by executing 'npm install loopback-connector-oracle --save' which will internally execute 'npm install oracledb' installing oracledb node module successfully. 
+
+Troubleshooting Tip: If you have problem installing loopback-connector-oracle make sure these env variables are set and these symbolic links exist. For e.g on OS X
+
+```
+export OCI_HOME=$HOME/instantclient_11_2
```

Comment:
- Isn't OCI_XXX variables are set only when user installs oracle client libraries directly from https://github.com/oracle/node-oracledb/blob/master/INSTALL.md#instosx not when their installation goes through with our prebuild tar ball? In that case, if user has followed instructions based on https://github.com/oracle/node-oracledb/blob/master/INSTALL.md#instosx, their directory is export $HOME/instantclient_11_2 not $HOME/oracle-instant-client